lib/api.js

```diff
@@ -742,6 +743,9 @@ API.prototype._getHeaders = function(method, url, args) {
   var headers = {
     'x-client-version': 'bwc-' + Package.version,
   };
+  if (this.supportStaffWalletId) {
```

Comment:
This is set externally from the command line tool. Do you think we should provide a method `setSupportStaffWalletId()` instead?